.github/CHANGELOG.md

```diff
@@ -224,6 +224,30 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 * PennyLane's test suite is now code-formatted using `black -l 100`.
   [(#1222)](https://github.com/PennyLaneAI/pennylane/pull/1222)
 
+* Adds a `hamiltonian_expand` tape transform, which takes a tape ending in 
+  `qml.expval(H)`, where `H` is a Hamiltonian and maps it to a collection 
+  of tapes which can be executed, and passed into a post-processing function yielding 
+  the expectation value.
+  [(#1142)](https://github.com/PennyLaneAI/pennylane/pull/1142)
+   
+  Example use:
+    
+  ```python
+  H = qml.PauliZ(0) + 3 * qml.PauliZ(0) @ qml.PauliX(1)
```

Comment:
Just a suggestion: is it more informative to show a QNode evaluation? I feel that what the users will most want to know is that they can now use `qml.expval(H)` inside a QNode. This is not so clear from the changelog description